README.md

```
@@ -16,15 +16,22 @@ It has the following format and structure:
 | `cn` | string | Chinese player analysis translation of the Hilichurlian word |
 | `notes` | string | Notes and additional information about the Hilichurlian word |
 
+### CLI Available
+
+> **Run via npx (no installation required)**
+>   - Requirements: NodeJS LTS v24.11.0 or later
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)